lib/game_runner.rb

```diff
@@ -20,7 +20,7 @@ def start
       loop do
         game_option = ui.menu(GAME_OPTIONS)
         player1, player2 = player_factory.create_players(game_option)
-        game.play(player1, player2)
+        game.play([player1, player2], Board.new)
```

Comment:
If you put player1 and player2 back into a list anyway, you don't need to the 
`player1, player2 = player_factory...` thing. 
Just use the array and call the variable `players`.